api/config/config_local.json

```diff
@@ -89,7 +89,6 @@
     "unsafeAllowDeleteUser": true,
     "enableVpcFlowLogs": true,
     "enableVpcServicePerimeter": true,
```

Comment:
This looks like a bad rebase or merge - please fix. Or else you need to rebase this PR onto master.

Reply:
Ah I see - it is based on a different branch - reviewing that now.

Reply:
Update 2: OK - that one is already merged. In that case - please update this PR to be based on master instead of that feature branch.

Reply:
Oh wow, you can change the base without closing the PR. That's really handy.